docs/UsersGuide.asciidoc

```diff
@@ -639,6 +652,26 @@ openqa-client isos post ISO=my_iso.iso DISTRI=my_distri FLAVOR=sweet \
 --------------------------------------------------------------------------------
 
 
+=== Job template YAML ===
+
+Job groups can be queried via the experimental REST API:
+
+    api/v1/experimental/job_templates_scheduling
+
+A GET request will get the YAML for one or multiple groups.
+
+[source,sh]
+--------------------------------------------------------------------------------
+./scripts/dump_templates --group test > test.yaml
```

Comment:
Please reference the in-path applications, not relative paths to the source folder of openQA. For openSUSE we can simply write "openqa-dump-templates". I feel though we should move the symlink creation to the Makefile …

It's a bit weird that you describe GET (and further below POST) but then just call other scripts instead. I would at least mention that dump_templates does a GET internally when you mention it as an example